README.md

```diff
@@ -308,6 +308,132 @@ for vehicle in vehicle_path_list:
+For instance, you can:
+ * Get its status;
+ * Get the machine type where it ran/is running;
+ * Kill it if you've changed your mind;
+ * Download output files (all of them or only the important ones);
+ * Get its execution time.
+
+Check out some example usage of `Task` after submitting an asynchronous simulation:
+
+```python
+# `scenario` constructed as in the examples above.
+# ... stands for arguments related to each specific scenario.
+task = scenario.simulate(..., run_async=True)
+
+# Get status of the task.
+status = task.get_status()
+print(status) # would be, e.g., "submitted", "started", "success", "failed", "killed"
+
+# Kill a task that hasn't completed yet.
+if status == "submitted" or status == "started":
+    task.kill()
+```
+
+```python
+
+task = scenario.simulate(..., run_async=True)
+
+# Get the output of the task. If the task is still running, this will block
+# until the outputs are ready.
+# `output` objects vary depending on the scenario, exposing methods to
+# manipulate the results that are relevant for that scenario!
+output = task.get_output()
+
+# Alternatively, download the raw output files!
+
+# If you don't know the files generated by your task, you can get an object
+# representing information on the output files.
+outputs_info = task.get_output_files_info()
+
+full_download_size = outputs_info.size # Size of the full output archive.
+files = outputs_info.contents # List of individual files available.
+
+# Pretty print information on the output archive.
+print(outputs_info)
+# Console logs:
+#
+# Archive size: 1.64 GiB
+# Contents:
+#  Size         Compressed   Name
+#  191 B        96 B         stderr.txt
+#  138.24 KiB   16.73 KiB    stdout.txt
+# ...
+#  3.80 MiB     3.61 MiB     important_file1.txt
+#  1.27 MiB     1.21 MiB     important_file2.txt
+
+
+# Download only those you are interested in:
+output_dir = task.download_outputs(
+    filenames=["important_file1.txt", "important_file2.txt"])
+# Console logs:
+#
+# 100%|██████████| 4.82M/4.82M [00:00<00:00, 273MiB/s]
+
+# Or all the generated files for archival, without extracting the
+# downloaded zip.
+output_dir = task.download_outputs(uncompress=False)
+# Console logs:
+#
+# 100%|██████████| 1.64G/1.64G [00:32<00:00, 55.1MiB/s]
+```
+
+### Retrieving tasks from previous sessions
+
+The real usefulness of the API arises from its ability to run long tasks asynchronously. Of course, this is not handy if you need to keep your Python session on to persist the task objects representing async running
+simulations!
+You can retrieve tasks and reconstruct the `Task` objects of previously created tasks using the `inductiva.tasks.get()` function.
+It requires an argument named `last_n`, which specifies the number of most recent tasks submitted to the API to retrieve. It returns a list of `Task` objects -- the same objects that you get from the call to `scenario.simulate()` -- so that you can resume manipulating the task and its results.
```

Comment:
The thing between -- --  should probably be on top when you mention Task objects for the first time again.